transformer_engine/pytorch/module/layernorm_mlp.py

```diff
@@ -201,10 +195,24 @@ def forward(
                     columnwise=(is_grad_enabled and fc1_weight.requires_grad),
                 )
 
+        ub_obj_lnout = None
+        ln_out = None
+        if ub_overlap_ag:
+            ub_obj_lnout = get_ub("fc1_fprop")
+            ln_out = ub_obj_lnout.get_buffer(fc1_input_quantizer, True)
+        elif with_quantized_norm:
+            ln_out = fc1_input_quantizer.make_empty(
+                inputmat.shape, dtype=inputmat.dtype, device="cuda"
+            )
+        else:
+            ln_out = torch.empty_like(
+                inputmat, dtype=inputmat.dtype, memory_format=torch.contiguous_format
+            )
+
         # Apply normalization
-        ln_out, mu, rsigma = apply_normalization(
+        _, mu, rsigma = apply_normalization(
             inputmat,
-            None,
+            ln_out,
```

Comment:
Same as https://github.com/NVIDIA/TransformerEngine/pull/1427#discussion_r1932936449:
```suggestion
        ub_obj_lnout = None
        ln_out = None
        if ub_overlap_ag:
            ub_obj_lnout = get_ub("fc1_fprop")
            ln_out = ub_obj_lnout.get_buffer(fc1_input_quantizer, True)

        # Apply normalization
        ln_out, mu, rsigma = apply_normalization(
            inputmat,
            ln_out,
```